core/src/main/web/app/mainapp/components/notebook/codecell-directive.js

```diff
@@ -417,7 +417,8 @@
               //codecomplete is up, skip
               return;
             }
-            scope.cm.execCommand('goDocStart');
+
+            scope.cm.setSelection({line: 0, ch: 0 }, {line: 0, ch: 0 }, {scroll: false});
```

Comment:
This command  scope.cm.execCommand('goDocStart'); calls  scope.cm.setSelection. See documentation about this function

doc.setSelection(anchor: {line, ch}, ?head: {line, ch}, ?options: object)
Set a single selection range. anchor and head should be {line, ch} objects. head defaults to anchor when not given. These options are supported:
scroll: boolean
Determines whether the selection head should be scrolled into view. Defaults to true.

We get scrolling on a big code cells and problem with focusing as result